src/Adapter/Product/Combination/Update/CombinationStockProperties.php

```diff
@@ -88,9 +88,9 @@ public function __construct(
     /**
      * @return int|null
      */
-    public function getQuantity(): ?int
+    public function getDeltaQuantity(): ?int
```

Comment:
I guess no BC break because class not released in 1.7.8.0 ?

Reply:
BC break to document, but acceptable one